app/presenters/curation_concerns/inspect_work_presenter.rb

```diff
@@ -0,0 +1,64 @@
+module CurationConcerns
+  class InspectWorkPresenter
+    def initialize(solr_document, current_ability)
+      @solr_document = solr_document
+      @current_ability = current_ability
+    end
+    attr_reader :solr_document, :current_ability
+
+    def workflow
+      if sipity_entity
+        {
+          entity_id: sipity_entity.id,
+          proxy_for: sipity_entity.proxy_for_global_id,
+          workflow_id: sipity_entity.workflow_id,
+          workflow_name: sipity_entity.workflow_name,
+          state_id: sipity_entity.workflow_state_id,
+          state_name: sipity_entity.workflow_state_name,
+          comments: workflow_comments,
+          roles: sipity_role # TODO: How to get related roles and actors for the sipity_entity
+        }
+      end
+    end
+
+    def solr
+      work.to_solr if work
```

Comment:
@jcoyne Here was my thinking, though my implementation is certainly lacking: When I'm running down a problem (in general, not just workflow related), I'm usually in the console asking 3 things: what does the object look like in Fedora, what is it _supposed_ to look like if I were to solrize it right now, and what does it really look like in Solr right now? When there's a problem, those things may not be as expected and can show where the problem lies. This view would shortcut that process in a single GET but I'm fine to just avoid for now if it isn't clean or useful.